### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by veselink1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,197 @@
-# manyclangs
-Repository hosting unofficial binary pack files for many commits of llvm
+# manyclangs <img src="https://llvm.org/img/DragonSmall.png" width="auto" height="72">
+Repository hosting unofficial [elfshaker](https://github.com/elfshaker/elfshaker) packs for many commits of LLVM.
+
```
Comment (peterwaller-arm) [inline README.md:5]: I'd like to see something visible at the top which makes the value proposition or shows 'the magic' clear. The emphasis in this sentence basically requires someone to go and read about elfshaker before they can understand what it is.

How about something like:
> manyclangs is a project enabling you to run any commit of clang within a few seconds, without having to build it.

Then we can continue to draw the readers in from there (with a few more sentences if needed).

Comment (peterwaller-arm) [reply README.md:5]: 👍

**Review by peterwaller-arm — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,199 @@
...
+- [2. Install manyclangs pack files](#2-install-manyclangs-pack-files)
+- [3. Run LLVM tools](#3-run-llvm-tools)
+- [More on linking](#more-on-linking)
+- [Running manyclangs on other platforms](#running-manyclangs-on-other-platforms)
+- [Our build configuration](#our-build-configuration)
+- [See also](#see-also)
+
+You need an elfshaker repository to extract the binaries into. An elfshaker repository is simply a directory with an `elfshaker_data` subdirectory.
+
+### 1. Create a repository for manyclangs
```
Comment (peterwaller-arm) [inline README.md:60]: I wonder if we want a summarized 'quickstart' which just has a set of stuff you can paste to get a build.

(including wget of a pack, and a specified build, for example).

Comment (veselink1) [reply README.md:60]: Would that include setting up the docker image?

Comment (peterwaller-arm) [reply README.md:60]: Maybe? What I have in mind is 'Whatever it takes for the user to run a specific clang commit, in one code block, including getting the dependencies.', if that's easy to express.

Comment (peterwaller-arm) [reply README.md:60]: (It's also good that you already go into detail about what it does later)

Comment (veselink1) [reply README.md:60]: We need to make the repo public for me to test that the script works.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,238 @@
...
+## Quickstart
+
+### AArch64
+Requirements: `elfshaker`, `wget`, `git`
+```bash
+mkdir -p manyclangs/elfshaker_data/packs
+YYYYMM=202109
+wget https://github.com/elfshaker/manyclangs/releases/download/v0.9.0/aarch64-ubuntu2004-manyclangs-$YYYYMM.pack{,.idx} -P manyclangs/elfshaker_data/packs
+cd manyclangs
+elfshaker find ''
```
Comment (peterwaller-arm) [inline README.md:17]: `find ''`?

Comment (peterwaller-arm) [reply README.md:17]: Oh, right, it is going to match all snapshots. Nice. Surprising somehow, but OK :)

Comment (veselink1) [reply README.md:17]: No, it is on purpose. We've discussed it before.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,238 @@
...
+```
+
+### x86-64
+Requirements: `elfshaker`, `wget`, `git`, `docker`
+
+```bash
+mkdir -p manyclangs/elfshaker_data/packs
+YYYYMM=202109
+wget https://github.com/elfshaker/manyclangs/releases/download/v0.9.0/aarch64-ubuntu2004-manyclangs-$YYYYMM.pack{,.idx} -P manyclangs/elfshaker_data/packs
+wget https://raw.githubusercontent.com/elfshaker/manyclangs/docker-qemu-aarch64/docker-qemu-aarch64/Dockerfile
```
Comment (peterwaller-arm) [inline README.md]:
```suggestion
wget https://raw.githubusercontent.com/elfshaker/manyclangs/master/docker-qemu-aarch64/Dockerfile
```

On the master branch, or is there a good reason to keep it separate?

**Review by peterwaller-arm — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (veselink1) [reply README.md]: Should be main.

File: README.md (hunk 3 above)
Comment (peterwaller-arm) [reply README.md:17]: It just looks surprising, I think, in user interface terms. I had forgotten, and wondered if it was the document rendering incorrectly somehow, for a moment.

**Pull request merged**